### PR TITLE
DXF-2655 Add spring.config.additional-location system property

### PR DESCRIPTION
**Pull request by vstasinopoulos**

### CODE REVIEW

File: yilu-common/templates/_deployment.yaml [hunk 1]
```diff
@@ -77,6 +77,8 @@ spec:
           value: 'true'
 {{- end }}
 {{- if .Values.secretsEnabled }}
+        - name: SPRING_CONFIG_ADDITIONAL-LOCATION
```
Comment (carlosrogue) [inline yilu-common/templates/_deployment.yaml]: Duplicated code:

https://github.com/yiluhub/common-chart/blob/bea9d62b824abb90e40e5c5cb2d0efe8a6936e65/yilu-common/templates/_deployment.yaml#L41-L43

Comment (carlosrogue) [reply yilu-common/templates/_deployment.yaml]: Also `SPRING_CONFIG_ADDITIONAL_LOCATION` instead?

Comment (vstasinopoulos) [reply yilu-common/templates/_deployment.yaml]: Duplicate yes. I am reverting it now... value is correct. For more info please see [here](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html)

Comment (carlosrogue) [reply yilu-common/templates/_deployment.yaml]: Yes, I forgot the Spring parsing, so, why not instead setting it as `spring.config.additional-location` ? It's more clear, I don't know, hehehehehe

Comment (haggucan) [reply yilu-common/templates/_deployment.yaml]: I think @carlosrogue  is mentioning the `-` between `ADDITIONAL` and `LOCATION `  that it should be `_` as well maybe by mistake but parameter is normally `spring.config.additional-location` that's why I think it was defined in that way

Comment (carlosrogue) [reply yilu-common/templates/_deployment.yaml]: Yes, the difference between what I expect from an actual `ENVIRON_VARIABLE` and a spring property (set as environment variable)

Comment (vstasinopoulos) [reply yilu-common/templates/_deployment.yaml]: I completely agree with you. It was also confusing for me.

**Pull request closed without merging**